README.md

```diff
@@ -21,4 +21,5 @@ and the lines starting with `*` are automatically rendered as bullet points.
 * Arna Ghosh [arnaghosh](http://github.com/arnaghosh)
 * Harsh Gupta [hargup](http://github.com/hargup)
 * Krishna Bagadia [krishna95](http://github.com/krishna95)
+* Manash Pratim Das	[ManashRaja](https://github.com/ManashRaja)
```

Comment:
You have inserted a tab between your name and the github handle, replace it by a space.